src/OrderFormAcceptanceTests.Tests/Features/CreateAnOrder.feature

```diff
@@ -0,0 +1,21 @@
+﻿Feature: Create An Order
+	As a Buyer
+	I want to create an Order
+	So that I can order from the Buying Catalogue
+
+Background: 
+	Given that a User is not authenticated and the user logs in using the 'login' function
+	When the User is a Buyer User
+	Then the User will be logged in
```

Comment:
I think you should take the opportunity to create a common step which wraps all this up and puts the user on the orderform dashboard

Reply:
Good idea, see next commit

Reply:
Maybe you want to use your new step "that a buyer user has logged in"